connector/connect/common/src/main/protobuf/spark/connect/storage_level.proto

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Should we rename this to something else at a high level? e.g., common.proto or utils.proto. Otherwise, LGTM from my end.

cc @hvanhovell @grundprinzip @ueshin

Reply:
Yes, I also think there should be a high level name, let me change it to ·common.proto· first. If others have better suggestions, I will rename it again

Reply:
I found that `connect-gen-protos.sh` won't automatically delete `storage_level_pb2.py` & `storage_level_pb2.pyi`, hahaha

Reply:
`common.proto` sounds good to me.